ntbs-service/Pages/Migration/Index.cshtml.cs

```diff
@@ -19,12 +25,21 @@ public IndexModel(INotificationImportService service)
         {
             _service = service;
             Results = new List<Notification>();

```

Comment:
Not added in this changeset, but can just add a default value for the property and leave it out of the constructor e.g. `public List<x> y {get; set;} = new List<x>();`